examples/util/tflite/TFliteModelImporter.js

```diff
@@ -133,7 +133,43 @@ class TFliteModelImporter {
     }, new Float32Array(tensor));
   }
 
-  _addOpsAndParams() {
+  async * layerIterator(inputTensors) {
+    const graph = this._rawModel.subgraphs(0);
+    const operatorsLength = graph.operatorsLength();
+    for (let lastNode = 0; lastNode < operatorsLength; ++lastNode) {
+      this._tensorIds = [];
+      this._operands = [];
+      this._operandIndex = 0;
+      if (this._backend !== 'WebML' && this._compilation) {
+        this._compilation._preparedModel._deleteAll();
+      }
+
+      this._model = await this._nn.createModel({backend: this._backend});
+      this._addTensorOperands();
+      lastNode = this._addOpsAndParams(lastNode);
+
+      const operator = graph.operators(lastNode);
+      const opCode = this._rawModel.operatorCodes(operator.opcodeIndex()).builtinCode();
+      const opcodeName = tflite.BuiltinOperator[opCode];
+
+      const inputs = Array.from(graph.operators(0).inputsArray());
```

Comment:
It would add operator inputs, like weights and bias into the inputs. It should use graph input `graph.inputsArray()`.